roles/monitoring/tasks/main.yml

```diff
@@ -0,0 +1,60 @@
+- name: Enable extras repo
+  command: subscription-manager repos --enable=rhel-7-server-extras-rpms
+  when: inventory_hostname in groups.loadbalancers
+- name: Install docker
+  yum:
+    name: docker
+    state: latest
+  when: inventory_hostname == 'localhost' or inventory_hostname in groups.loadbalancers
+  become: yes
+  become_method: sudo
+- name: Enable and start docker daemon service
+  systemd:
+    name: docker
+    state: started
+    enabled: yes
+    daemon_reload: yes
+  when: inventory_hostname == 'localhost' or inventory_hostname in groups.loadbalancers
+  become: yes
+  become_method: sudo
+- name: create persistent storage volume for zabbix
```

Comment:
Is this storage on ceph? If not then it might be an idea to move it there as this is a single node deployment?

Reply:
No its not - that's something that will require a bit more work, I suggest we raise an issue? Alternatively moving all the nodes in heat to be volume backed instead of ephemeral might be an idea?

Reply:
Yeah I guess all nodes moving will bump the costs? What about just moving the bastion in HEAT. Is it declared separately? If not then I agree, lets raise an issue.

Reply:
It will, we could do the bastion independently though in the heat template, but that work will need to go on the backlog whilst the other higher priorities are completed.